dpdata/plugins/fhi_aims.py

```diff
@@ -4,7 +4,7 @@
 @Format.register("fhi_aims/md")
 @Format.register("fhi_aims/output")
 class FhiMDFormat(Format):
-    def from_labeled_system(self, file_name, md=True, begin = 0, step = 1, **kwargs):
+    def from_labeled_system(self, file_name, md=True, begin = 0, step = 1, converge=True, **kwargs):
```

Comment:
I would suggest using `req_converge` or `conv_check` in the interface, because it is more self-explanary. 
The same suggest for the pwmat and vasp interfaces.

Reply:
All are denoted by `convergence_check`.